src/api/config/options.yml.example

```diff
@@ -183,6 +183,14 @@ default: &default
   #    persistent: true
   #    passive: true
 
+  # For storing rails metrics in influxdb
+  # https://github.com/influxdata/influxdb-rails
+  # influxdb_database: rails
+  # influxdb_username:
+  # influxdb_password:
+  # influxdb_hosts:
```

Comment:
Please fill those with example data, so people at least have the chance to know what is expected.